web/src/components/overview/StorageSection.jsx

```diff
@@ -155,7 +155,7 @@ export default function StorageSection() {
     const pvDevices = result.settings.targetPVDevices;
 
     if (pvDevices.length > 1) {
-      return <span>{msgLvmMultipleDisks(result.settings.spacePolicy)}</span>;
+      return <Content><span>{msgLvmMultipleDisks(result.settings.spacePolicy)}</span></Content>;
```

Comment:
Most probably we should refactor this component for addressing the problem in a better, more elegant way. But let's fixed it in the simpler way by now.